packages/garbo/src/resources/doghouse.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  $item,
+  $location,
+  get,
+  haveInCampground,
+  realmAvailable,
+} from "libram";
+import { globalOptions } from "../config";
+import { garboValue } from "../garboValue";
+import { mallPrice } from "kolmafia";
+import { hotTubAvailable } from "./clanVIP";
+
+export function lavaDogsAccessible(): boolean {
+  return (
+    haveInCampground($item`haunted doghouse`) &&
+    !get("doghouseBoarded") &&
+    realmAvailable("hot")
+  );
+}
+
+export function lavaDogsComplete(): boolean {
+  return (
+    get("hallowienerVolcoino") ||
+    $location`The Bubblin' Caldera`.turnsSpent >= 7 ||
+    $location`The Bubblin' Caldera`.noncombatQueue.includes("Lava Dogs")
+  );
+}
+
+export function shouldLavaDogs(): boolean {
+  return (
+    globalOptions.ascend &&
+    lavaDogsAccessible() &&
+    garboValue($item`Volcoino`) >
+      7 * get("valueOfAdventure") +
```

Comment:
I don't think 7 is right? I'm pretty sure the ghost dog adventure is free. I can check in a sec.

Reply:
I zoned out and did not check.

Reply:
Having done some spading, _this_ one should be 6